src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -915,7 +916,8 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter)
       !intp.reporter.hasErrors
     }
     loadFiles(settings)
-    printWelcome()
+    if( ! Properties.propOrFalse( "scala.repl.disableWelcomeMessage" ) ) 
+      printWelcome()
```

Comment:
Thank you for comment. 
Haven't spotted ReplProps before. 
It is a bit unfortunate that configs are in different namespaces "scala.shell" vs "scala.repl". 

Things about string evaluation are maybe too strong for me in this stage... :-)